README.md

```diff
@@ -11,6 +11,15 @@ Flatpak.
 Then build via
 
 ```
+flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo
+flatpak remote-add --if-not-exists flathub-beta https://flathub.org/beta-repo/flathub-beta.flatpakrepo
+flatpak -y uninstall tv.kodi.Kodi
+flatpak -y install org.flatpak.Builder/x86_64/stable
+flatpak -y install org.freedesktop.Platform/x86_64/24.08
+cd
```

Comment:
You can't assume peoples file structures, please remove this and the `rm` in the next line